frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnField.scss

```diff
@@ -18,5 +18,9 @@
     line-height: var(--pf-global--spacer--xl);
     position: absolute;
     right: -16px;
+    &.is-disabled {
+      pointer-events: none;
+      color: var(--pf-global--disabled-color--100);
```

Comment:
should be `--pf-global--disabled-color--200` to align with the plain button variant.

I created a tech debt item to convert this delete button to an actual button.

Reply:
updated to ```--pf-global--disabled-color--200```